pkg/clipboard/clipboard.go

```diff
@@ -16,27 +17,58 @@ var (
 	// -ldflags=='-X github.com/gopasspw/gopass/pkg/clipboard.Helpers=termux-api'.
 	Helpers = "xsel or xclip"
 	// ErrNotSupported is returned when the clipboard is not accessible.
-	ErrNotSupported = fmt.Errorf("WARNING: No clipboard available. Install " + Helpers + " or use -f to print to console")
+	ErrNotSupported = fmt.Errorf("WARNING: No clipboard available. Install " + Helpers + ", provide $GOPASS_CLIPBOARD_COPY_CMD and $GOPASS_CLIPBOARD_CLEAR_CMD or use -f to print to console")
 )
 
 // CopyTo copies the given data to the clipboard and enqueues automatic
 // clearing of the clipboard.
 func CopyTo(ctx context.Context, name string, content []byte, timeout int) error {
-	if clipboard.Unsupported {
-		out.Printf(ctx, "%s", ErrNotSupported)
-		_ = notify.Notify(ctx, "gopass - clipboard", fmt.Sprintf("%s", ErrNotSupported))
-		return nil
-	}
+	clipboardCopyCMD := os.Getenv("GOPASS_CLIPBOARD_COPY_CMD")
+	if clipboardCopyCMD != "" {
+		clipboardProcess := exec.Command(clipboardCopyCMD, name)
+		stdin, err := clipboardProcess.StdinPipe()
+		defer stdin.Close()
+
+		if err != nil {
+			_ = notify.Notify(ctx, "gopass - clipboard", "failed to open STDIN of clipboard copy process")
+			return fmt.Errorf("failed to open STDIN of clipboard copy process: %w", err)
+		}
+		if err = clipboardProcess.Start(); err != nil {
+			_ = notify.Notify(ctx, "gopass - clipboard", "failed to start clipboard copy process")
+			return fmt.Errorf("failed to start clipboard copy process: %w", err)
+		}
+		if _, err = stdin.Write(content); err != nil {
+			_ = notify.Notify(ctx, "gopass - clipboard", "failed to write to STDIN of clipboard copy process")
+			return fmt.Errorf("failed to write to STDIN of clipboard copy process: %w", err)
+		}
+
+		// Force STDIN close before we wait for the process to finish, so we avoid deadlocks
+		if err = stdin.Close(); err != nil {
+			_ = notify.Notify(ctx, "gopass - clipboard", "failed to close STDIN of clipboard copy process")
+			return fmt.Errorf("failed to close STDIN of clipboard copy process: %w", err)
+		}
+
+		if err := clipboardProcess.Wait(); err != nil {
+			_ = notify.Notify(ctx, "gopass - clipboard", "failed to execute $CLIPBOARD_COPY_CMD")
+			return fmt.Errorf("failed to execute $CLIPBOARD_COPY_CMD: %w", err)
+		}
+	} else {
+		if clipboard.Unsupported {
+			out.Printf(ctx, "%s", ErrNotSupported)
```

Comment:
This should probably be a `out.Errorf` instead.

Reply:
I can adjust it, but please note that I didn't change this.

Reply:
This could also solve the below issue about the integration test not working anymore.

Reply:
It won't – the test currently looks at `cmd.CombinedOutput()`, which combines `STDOUT` and `STDERR`.

Reply:
Fixed in 7ffa1037